src/NzbDrone.Core/Parser/Parser.cs

```diff
@@ -761,6 +761,29 @@ public static string RemoveFileExtension(string title)
             return title;
         }
 
+        public static string ToUrlSlug(string value)
```

Comment:
I don't think this should be part of the parser, `StringExtensions` would be a better place.

Reply:
I would agree. I've added it in Parser as thats where the same function is in Radarr. Let me know if you still want me to move it.

Reply:
Yeah, let's move it.